sway.el

```diff
@@ -501,6 +507,21 @@ argument for the undertaker.."
 ;;(let ((process-environment (frame-parameter frame 'environment)))
 ;;(call-process sway-swaymsg-binary nil nil nil (format sway-focus-message-format focused))))
 
+;;; Embark
+
+(defvar sway-container-embark-actions
+  (let ((map (make-sparse-keymap)))
+    (define-key map "k" #'sway-kill-container)
+    (define-key map "f" #'sway-focus-container)
+    map)
+  "Keymap for actions for Sway container items.")
+
+(defvar embark-keymap-alist)
```

Comment:
Do we need this?

Reply:
Yes, since we don't (require 'embark) (and it's better not to IMO, so that the package works for those without), this is useful for byte-compilation, so that the variable is defined.